src/NServiceBus.ContainerTests/NServiceBus.ContainerTests.csproj

```diff
@@ -1,86 +1,37 @@
-    <Compile Include="When_using_nested_containers.cs" />
-    <Compile Include="When_building_components.cs" />
-    <Compile Include="When_querying_for_registered_components.cs" />
-    <Compile Include="When_registering_components.cs" />
+    <ProjectReference Include="..\NServiceBus.Core\NServiceBus.Core.csproj" />
   </ItemGroup>
+
   <ItemGroup>
-    <ProjectReference Include="..\NServiceBus.Core\NServiceBus.Core.csproj">
-      <Project>{DD48B2D0-E996-412D-9157-821ED8B17A9D}</Project>
-      <Name>NServiceBus.Core</Name>
-    </ProjectReference>
-  </ItemGroup>
-  <ItemGroup>
-    <None Include="packages.config" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.2.0" PrivateAssets="All" />
+    <PackageReference Include="NUnit" Version="[3.12.0, 4.0.0)" />
+    <PackageReference Include="NUnit3TestAdapter" Version="3.15.1" PrivateAssets="All" />
+    <PackageReference Include="Particular.Packaging" Version="0.5.0" PrivateAssets="All" />
   </ItemGroup>
+
+  <PropertyGroup>
+    <PackageId>NServiceBus.ContainerTests.Sources</PackageId>
+    <Description>Container tests for nservicebus core functionality</Description>
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+  </PropertyGroup>
+
   <ItemGroup>
-    <Service Include="{82A7F48D-3B50-4B1E-B82E-3ADA8210C358}" />
+    <Compile Update="**\*.cs" Pack="true" PackagePath="content\App_Packages\NSB.ContainerTests\;contentFiles\cs\any\NSB.ContainerTests\" />
   </ItemGroup>
-  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
-  <Target Name="EnsureNuGetPackageBuildImports" BeforeTargets="PrepareForBuild">
-    <PropertyGroup>
-      <ErrorText>This project references NuGet package(s) that are missing on this computer. Use NuGet Package Restore to download them.  For more information, see http://go.microsoft.com/fwlink/?LinkID=322105. The missing file is {0}.</ErrorText>
-    </PropertyGroup>
-    <Error Condition="!Exists('..\packages\NUnit.3.11.0\build\NUnit.props')" Text="$([System.String]::Format('$(ErrorText)', '..\packages\NUnit.3.11.0\build\NUnit.props'))" />
-    <Error Condition="!Exists('..\packages\NuGetPackager.0.6.5\build\NuGetPackager.targets')" Text="$([System.String]::Format('$(ErrorText)', '..\packages\NuGetPackager.0.6.5\build\NuGetPackager.targets'))" />
-    <Error Condition="!Exists('..\packages\NUnit3TestAdapter.3.13.0\build\net35\NUnit3TestAdapter.props')" Text="$([System.String]::Format('$(ErrorText)', '..\packages\NUnit3TestAdapter.3.13.0\build\net35\NUnit3TestAdapter.props'))" />
-    <Error Condition="!Exists('..\packages\GitVersionTask.4.0.0-beta0013\build\GitVersionTask.targets')" Text="$([System.String]::Format('$(ErrorText)', '..\packages\GitVersionTask.4.0.0-beta0013\build\GitVersionTask.targets'))" />
+
+  <!-- Workaround for https://github.com/nunit/nunit3-vs-adapter/issues/697 -->
+  <Target Name="RemoveNUnit3TestAdapterFromPackage" BeforeTargets="_GetPackageFiles">
```

Comment:
https://github.com/nunit/nunit3-vs-adapter/issues/697 seems fixed but not released, is it worth waiting for the fix?

Reply:
I'd say no. We can always update and remove the workaround when it's released.